client/components/foldable-card/index.jsx

```diff
@@ -96,6 +96,10 @@ var FoldableCard = React.createClass( {
 	},
 
 	renderContent: function() {
+		if ( ! this.state.expanded ) {
```

Comment:
I think it'd be a bit more readable to do `{ this.state.expanded && this.renderContent() }` in the main render function.